string source_frame_id
string target_frame_id
sensor_msgs/PointCloud pcl_in
---
sensor_msgs/PointCloud pcl_out